Tests/NIOTests/ByteBufferTest.swift

```diff
@@ -1073,22 +1073,37 @@ class ByteBufferTest: XCTestCase {
     }
 
     func testAllocationOfReallyBigByteBuffer() throws {
+        #if !arch(arm) // !32-bit, Raspi/AppleWatch/etc
```

Comment:
Can we invert this conditional? It'd be nice to see the intended effect of the other branch closer to the top of the test.